changelog/changelog.ddoc

```diff
@@ -24,14 +24,14 @@ $(SMALL released $1, $2)
 $4
 )
 
-NIGHTLY_VERSION=
-$(DIVC version,
-$(P
-$(B $(LARGE $(LINK2 http://nightlies.dlang.org, Download D nightlies)))$(BR)
-$(SMALL $1)
-)
-$4
-)
```

Comment:
How about leaving this in but removing it later so that we don't break the _pre page interim?

Reply:
This should no longer be an issue.